src/content/peerconnection/audio/index.html

```diff
@@ -27,55 +26,69 @@
   <link href="//fonts.googleapis.com/css?family=Inconsolata" rel="stylesheet" type="text/css">
   <link rel="stylesheet" href="../../../css/main.css" />
   <link rel="stylesheet" href="css/main.css" />
-
 </head>
 
 <body>
-
-  <div id="container">
-
-    <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>Peer connection: audio only</span></h1>
-
-    <div id="audio">
-      <div>
-        <div class="label">Local audio:</div><audio id="audio1" autoplay controls muted></audio>
-      </div>
-      <div>
-        <div class="label">Remote audio:</div><audio id="audio2" autoplay controls></audio>
-      </div>
-    </div>
-
-    <div id="buttons">
-      <select id="codec">
-        <!-- Codec values are matched with how they appear in the SDP.
-        For instance, opus matches opus/48000/2 in Chrome, and ISAC/16000
-        matches 16K iSAC (but not 32K iSAC). -->
-        <option value="opus">Opus</option>
-        <option value="ISAC">iSAC 16K</option>
-        <option value="G722">G722</option>
-        <option value="PCMU">PCMU</option>
-      </select>
-      <button id="callButton">Call</button>
-      <button id="hangupButton">Hang Up</button>
+<div id="container">
```

Comment:
looks like you changed indent here?

Reply:
Yes please check that the indentation is correct, otherwise it's a mess to read.

Reply:
Sure will test it once and let you know everything okey.

Reply:
Everything is fine in-fact, no issues found. kindly check you can approve.

Reply:
but indent changes from 2 spaces to 0 spaces (for this and everything else) -- which makes the diff much harder to read
(and yes, indent has been inconsistent before)

Reply:
Please indent correctly, it should belong to the body tag.